src/domains.cpp

```diff
@@ -153,15 +155,15 @@ unsigned int check_local_entropy (dolfin::Function *cation,
   status = fasp_solver_dbsr_krylov_diag(&adaptA_fasp_bsr, &b_fasp, &solu_fasp, &itpar);
```

Comment:
To get this working, replace this line with
```
status = fasp_solver_dcsr_krylov_diag(&A_fasp, &b_fasp, &solu_fasp, &itpar);
```
seems that the problem was with using a block solver.